modules/claims_api/app/controllers/concerns/claims_api/disability_compensation_validations.rb

```diff
@@ -587,5 +590,23 @@ def validate_form_526_direct_depost!
         )
       end
     end
+
+    def validate_form_526_at_least_one_active_duty_end_date_within_180_days!
+      service_periods = form_attributes.dig('serviceInformation', 'servicePeriods')
+
+      at_least_one_active_duty_end_date_within_180_days = service_periods.any? do |sp|
+        active_duty_end_date = sp['activeDutyEndDate']
+        next if active_duty_end_date.blank?
+
+        Date.parse(active_duty_end_date) <= 180.days.from_now
```

Comment:
This could change behavior based on time of day of the submission, which we likely don't want. Adding either `beginning_of_day` or `end_of_day`, whichever feels more appropriate; that should keep validation consistent regardless of the time-of-day of the submission.